gatsby-config.js

```diff
@@ -279,6 +281,61 @@ module.exports = {
         nodesPerFeedFile: Infinity,
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
```

Comment:
If I'm understanding this correctly, we only output a single RSS feed endpoint for this correct?

I see the requirements listed in the issue as this (emphasis mine):

> Similar to current site, each agent's (including mobile) release notes includes an RSS feed **listing the 15 most recent release notes**. Essentially **any folder** in docs/release-notes that contains release note files should have an RSS feed.

From my understanding, this means we should have one feed per release note type, and pull the 15 most recent releases from each. Am I understanding that correct? Or maybe the plan is to just consolidate to a single RSS feed?

Reply:
Oh good call, I'll update this to generate a few different feeds.

This plugin makes that fairly straightforward (via the `feeds` array). That said, I might extract this out to a plugin since it's already adding a lot of clutter to our config file.